app/assets/stylesheets/legacy_diff.scss

```diff
@@ -0,0 +1,80 @@
+// stylelint-disable selector-no-qualifying-type, max-nesting-depth
+
+// Diff of two editions
+
+$added-color: #ddffdd;
+$strong-added-color: #77f177;
+$removed-color: #ffdddd;
+$strong-removed-color: #ffaaaa;
+
+.diff {
+  border: 1px solid $gray-lighter;
+  border-left: 40px solid $gray-lighter;
+  border-radius: 3px;
+  padding: 15px;
```

Comment:
We should probably use the [Design System responsive spacing scale](https://design-system.service.gov.uk/styles/spacing/#responsive-spacing) variables for valuables like '40px' and '15px'. This would adapt to viewport sizes and allow for the values to potentially change in future keeping the layout consistent.